moveit_ros/manipulation/move_group_pick_place_capability/src/pick_place_action_capability.cpp

```diff
@@ -576,7 +472,6 @@ void move_group::MoveGroupPickPlaceAction::fillGrasps(moveit_msgs::PickupGoal &g
                                                  -std::numeric_limits<double>::max());
     }
     goal.possible_grasps.push_back(g);
```

Comment:
I'm pretty sure this section needs to be un-indented, but all my clang-format efforts have failed me since Travis didn't run